resources/views/partials/actions.blade.php

```diff
@@ -88,7 +88,21 @@ function drawChart($el) {
                             return "{{ color_darken($theme_reds, -0.1) }}";
                         }),
                         pointBorderColor: "{{ color_darken($theme_metrics, -0.1) }}",
-                        pointHoverBackgroundColor: "{{ color_darken($theme_metrics, -0.2) }}",
```

Comment:
I feel like this logic should be moved to the JS import and provide a function for registering. 

JS in the wild is ugly.